alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
```

Comment:
I wounder if that is the right default. The generic app might be better of in shipping a self signed certificate and only expose 443/https.